### ISSUE TITLE
Running Potree 8 on Plesk Server - Not Working

### ISSUE THREAD

**Issue by bmanwithab** [potree/potree#1135]
Hello all.  I have my own hosting server on Digital Ocean and it runs Plesk Obsidian.  I setup a subdomain to use Potree at.  When I "just copy the Potree 8" files over to the web server subdomain folder and then open a web browser, I get nothing.  Dev tab doesn't show any errors in Chrome or Firefox.  
So I looked in Plesk and it seems you have to install Nodejs on each domain.  I am stuck on that as directions for everything is pretty simple, but it seems it doesn't like the file structure in which Potree is built.  
I included some screenshots of the error.  I know enough to make it by on most things, but this is stumping me because of the lack of documents out there.  I didn't find anything that handles this situation.  I think I probably just need to restructure something but I am not sure what.  Any help is greatly appreciated.  Thanks.

![Screenshot 2021-12-28 160244](https://user-images.githubusercontent.com/24690096/147606841-429f3563-c1a3-4995-ae57-f87778d369fc.jpg)
![Screenshot 2021-12-28 160420](https://user-images.githubusercontent.com/24690096/147606957-7d0d3e90-f502-43c1-a523-1eaada77ea75.jpg)

**Comment by HeadLessHUN** [comment]
you don't need any nodejs app. just setup a webserver and copy over the release version of potree (libs, build, directories and then copy over the converted clouds to the pointcloud directory. last step is to create a html webpage for the given cloud...

**Comment by bmanwithab** [comment]
Yeah i can read the instructions too.  The problem is there is some kind of dependency i need to turn on, on the Plesk server.  
Your comment is of no help.  You are just repeating the instructions.  I setup a full self controlled webserver with a few clients, so i got the simple stuff.

**Comment by HeadLessHUN** [comment]
I am sorry that i couldn't help, but that is only the needed steps. If it is not working you should see around plesk instead, but firstly i would just disable nodejs it can be seen on your picture.

It does not require any dependency just the libs,build directory. no need for nodejs, for deployment. nodejs/gulp are only for building and development.

**Comment by midnight-dev** [comment]
Which web page did you create, and what was the error when going to that specific address on your server? 

I'm under the impression that you didn't try to make a project page and are not trying to access the existing ones. This isn't really an end-to-end deployable solution, so manual work is necessary. If you already made a page, we'd need a screenshot of the error you're seeing when visiting it, preferably with the console open.

Headless was right, by the way. The npm install is only relevant for dev work - bundling all of the JavaScript together and whatnot. There's no benefit to running it on your server and has no effect on the existing files you get from the repo.
